src/style.scss

```diff
@@ -219,7 +218,7 @@ h1.recipeTitle {
 .borderLeftRight {
```

Comment:
To coincide with changes made to `a.active` on line 113, **all** instances of `.borderLeftRight` should be changed to `.navbar-nav .borderLeftRight.nav-link`

Reply:
OK, I refactored all of the selectors that had .borderLeftRight to be under one parent selector and then changed that parent selector in the way you described.